app.js

```diff
@@ -1,9 +1,42 @@
-import * as config from './config/taskInfo.json';
 import logger from './config/logger';
+import convert from './src/utils/convertCsvToJson';
+import * as csvToJson from 'convert-csv-to-json';
+import * as config from './config/taskInfo.json';
+
 import { Product, User } from 'models';
 
+import Dirwatcher from './src/dirwatcher';
+import Importer from './src/importer';
 
 logger.info(config.name);
 
 const product = new Product();
 const user = new User();
+
+const watchModule = new Dirwatcher();
+const importModule = new Importer();
+
+watchModule.on('changed', (path) => { 
+    importModule.import(path).then(dataArr => {
```

Comment:
Also, a small notice is that according to task `import` method should be able to work with only 1 path, not with array